indra_network_search/__init__.py

```diff
@@ -0,0 +1,9 @@
+from configparser import ConfigParser
+from pathlib import Path
+
+fp = Path(__file__).absolute().parent.parent.joinpath("setup.cfg").as_posix()
+cfp = ConfigParser()
+cfp.read(fp)
+
+NAME = cfp.get('metadata', 'description')
```

Comment:
can also use https://docs.python.org/3/library/importlib.metadata.html

Reply:
Then I would have to cut py37, which I guess isn't really an issue

Reply:
there's a backport for it if you really love py37. but why not cut py38 while you're at it too 😈